admin.html

```diff
@@ -9,57 +9,58 @@
 // save index and prompts in the browser cache for refresh
 -->
 
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <title>BonoboConnect Admin</title>
-    <link href="admin.css" rel="stylesheet">
-    <link rel="icon" type="image/x-icon" href="bc_favicon.png">
+    <link href="prompt.css" rel="stylesheet" />
```

Comment:
was this intentional? are you trying to consolidate css files?  if so, we could delete admin.css